Mantle/MTLModel.m

```diff
@@ -232,7 +232,12 @@ + (MTLPropertyStorage)storageBehaviorForPropertyWithKey:(NSString *)propertyKey
 	};
 
 	if (attributes->readonly && attributes->ivar == NULL) {
-		return MTLPropertyStorageNone;
+		if ([self isEqual:MTLModel.class]) {
+			return MTLPropertyStorageNone;
+		} else {
+			// Check superclass in case the subclass redeclares a property that falls through
```

Comment:
Can you break this comment at 80 chars? I.e.

```objc
            // Check superclass in case the subclass redeclares a property that
            // falls through
```
?